.gitignore

```diff
@@ -1,7 +1,8 @@
 build
 .gradle
 .idea/
+*.dex
```

Comment:
Are dex files create in the root during the tests? Shouldn't they be created in a temp dir?